foreman/foreman.spec

```diff
@@ -557,7 +558,8 @@ Meta package to install requirements for Rackspace compute resource support.
 %package vmware
 Summary: Foreman VMware support
 Group:  Applications/System
-Requires: %{?scl_prefix}rubygem(fog-vsphere) >= 1.13.0
+Requires: %{?scl_prefix}rubygem(fog-vsphere) >= 2.3.0
+Requires: %{?scl_prefix}rubygem(fog-vsphere) < 2.4.0
```

Comment:
These need to be changed for 1,17.3 to >=2.1 <2.3 to make sure users get the known good version